index.html

```diff
@@ -16,8 +21,8 @@
           <header>
             <h1>{{title}}</h1>
             <div class="byline">
-              By <address><a href="{{authorUrl}}">{{author}}</a></address>
-              {{publishStatus}}
+              By <address><a href="{{authorUrl}}">{{author}}</a></address> {{publishStatus}}<br/>
+              <small>Read Time: {{readLength}}</small>
             </div>
           </header>
           <section class="article-body">{{{body}}}</section>
```

Comment:
You have lots of good elements within your code. I like that you added the read-on button. That is a nice feature and could be nice for more than just displaying extra parts of text. If you are showcasing photos from your recent projects you could just have the photo of the project and use the read on to show the description of the project.